src/extensions/default/bramble/lib/ConsoleManagerRemote.js

```diff
@@ -0,0 +1,76 @@
+(function(transport, global, console) {
+    "use strict";
+    
+    function transportSend(args, type) {        
```

Comment:
I'm not going to block you on changing this, but I find it surprising that you don't have the `type` first, and `args` second.  That's the opposite of what I'd expect.  Just a comment, in case your choice was random.